### PR TITLE
Update PublicApiClient.ts

### PR DESCRIPTION
**Pull request by Elli610**
related to #152

### CODE REVIEW

**Review by Ben-Rey — CHANGES_REQUESTED** [review]

File: src/web3/PublicApiClient.ts [hunk 1]
```diff
@@ -54,7 +72,12 @@ export class PublicApiClient extends BaseClient implements IPublicApiClient {
     }
   }
 
-  /** Get blockclique details by period and thread */
+  /**
+   * Get blockclique details by period and thread
+   *
+   * @param slot - The slot as an ISlot
+   * @return A promise which resolves in the blockclique details
```
Comment (0xSwapFeeder) [inline src/web3/PublicApiClient.ts:79]:
```suggestion
   *
   * @return A promise which resolves in the blockclique details
```

File: src/web3/PublicApiClient.ts [hunk 2]
```diff
@@ -89,7 +123,12 @@ export class PublicApiClient extends BaseClient implements IPublicApiClient {
     }
   }
 
-  /** Get info about a list of addresses (balances, block creation, ...) */
+  /**
+   * Get data about a list of addresses (balances, block creation, ...)
+   *
+   * @param addresses - The addresses as an array of strings
+   * @return A promise which resolves in the addresses data
```
Comment (0xSwapFeeder) [inline src/web3/PublicApiClient.ts:130]:
```suggestion
   *
   * @return A promise which resolves in the addresses data
```

File: src/web3/PublicApiClient.ts [hunk 3]
```diff
@@ -107,7 +146,12 @@ export class PublicApiClient extends BaseClient implements IPublicApiClient {
     }
   }
 
-  /** Show info about a block (content, finality ...) */
+  /**
+   * Show data about a block (content, finality ...)
+   *
+   * @param blockIds - The block ids as an array of strings
+   * @return A promise which resolves in the block data
```
Comment (0xSwapFeeder) [inline src/web3/PublicApiClient.ts:153]:
```suggestion
   *
   * @return A promise which resolves in the block data
```

File: src/web3/PublicApiClient.ts [hunk 4]
```diff
@@ -141,7 +190,12 @@ export class PublicApiClient extends BaseClient implements IPublicApiClient {
     }
   }
 
-  /** Show info about a list of operations = (content, finality ...) */
+  /**
+   * Show data about a list of operations
+   *
+   * @param operationIds - The operation ids as an array of strings
+   * @return A promise which resolves in the operation data
```
Comment (0xSwapFeeder) [inline src/web3/PublicApiClient.ts:197]:
```suggestion
   *
   * @return A promise which resolves in the operation data
```

File: src/web3/PublicApiClient.ts [hunk 5]
```diff
@@ -191,7 +253,12 @@ export class PublicApiClient extends BaseClient implements IPublicApiClient {
     }
   }
 
-  /** Returns the data entry both at the latest final and active executed slots. */
+  /**
+   * Returns the data entry both at the latest final and active executed slots.
+   *
+   * @param addressesKeys - The addresses and keys as an array of IDatastoreEntryInput objects
+   * @return A promise which resolves to the datastore entries
```
Comment (0xSwapFeeder) [inline src/web3/PublicApiClient.ts:260]:
```suggestion
   *
   * @return A promise which resolves to the datastore entries
```

**Review by 0xSwapFeeder — CHANGES_REQUESTED** [review]
Watch your ts doc tags !

**Review by Ben-Rey — APPROVED** [review]

**Review by gregLibert — APPROVED** [review]

**Review by 0xSwapFeeder — APPROVED** [review]

**Pull request merged**